### PR TITLE
Categorize .yaml files and initial train

### PR DESCRIPTION
**Pull request by quanglddev**
I figured out that dividing one big .yaml file containings many categories is hard to maintain so I change the script to make it split into many one-category .yaml file. 
Also, I use example code on ChatterBot website to try out the performance, not very well (please test)

### CODE REVIEW

**Review by ya332 — APPROVED** [review]

**Review by ya332 — APPROVED** [review]

File: scraper/Application Process - 1.yaml [hunk 1]
```diff
@@ -1,7 +1,5 @@
 categories:
 - Application Process
-- Scholarships and Financial Aid
```
Comment (ya332) [inline scraper/Application Process - 1.yaml:3]: How are you planning to train the ChattterBot read several yaml files ? Further investigation might be needed.

Comment (ya332) [reply scraper/Application Process - 1.yaml:3]: I saw your main.py and got my answer. I wonder if training it sequentially  on several different .yaml files makes any difference than training with one big yaml file at once?

Comment (quanglddev) [reply scraper/Application Process - 1.yaml:3]: It does make a difference (in a bad way). I still don't know why maybe I need to read more about how the training process works.

Comment (ya332) [reply scraper/Application Process - 1.yaml:3]: Initial training is successfully completed, but the bot is not robust. Small changes in the sentence results in very suboptimal outputs. Will open an issue to do some analysis.

Comment (quanglddev) [reply scraper/Application Process - 1.yaml:3]: Yes I believe after scraping the data we should also manually include variations of the question (manpower needed 😂)

**Pull request merged**